piker/data/_ahab.py

```diff
@@ -143,27 +143,37 @@ async def process_logs_until(
                 if not entry:
                     continue
 
+                entry = entry.strip()
                 try:
-                    record = json.loads(entry.strip())
+                    record = json.loads(entry)
+
+                    if 'msg' in record:
+                        msg = record['msg']
+                    elif 'message' in record:
+                        msg = record['message']
+                    else:
+                        raise KeyError('Unexpected log format')
+
+                    level = record['level']
+
                 except json.JSONDecodeError:
-                    if 'Error' in entry:
-                        raise RuntimeError(entry)
-                    raise
+                    # if 'Error' in entry:
+                    #     raise RuntimeError(entry)
+                    # raise
+                    msg = entry
+                    level = 'error'
 
-                msg = record['msg']
-                level = record['level']
                 if msg and entry not in seen_so_far:
                     seen_so_far.add(entry)
                     if bp_on_msg:
                         await tractor.breakpoint()
 
-                    getattr(log, level, log.error)(f'{msg}')
+                    getattr(log, level.lower(), log.error)(f'{msg}')
 
-                    # print(f'level: {level}')
-                    if level in ('error', 'fatal'):
+                    if level == 'fatal':
```

Comment:
my only concern is if this is gonna cause benign errors triggering crashes with `marketstore`?

@guilledk you tested this with the `--tsdb` stuff?

Reply:
I did a manual test starting with both `--es` and `--tsdb` and both opened ok, didnt do more extensive testing than that tho.

Reply:
Tests for initialization of `elastic` & `marketstore` are passing.